dkron/api.go

```diff
@@ -357,14 +362,55 @@ func (h *HTTPTransport) executionsHandler(c *gin.Context) {
 		return
 	}
 
+	if output_size > 0 {
+		// truncate execution output
+		for _, execution := range executions {
+			_s := len(execution.Output)
```

Comment:
Why the leading `_` in `_s` ? Why not simply use `size` as the variable name ?

Reply:
✅